packages/core/stencil.config.ts

```diff
@@ -6,6 +6,103 @@ import { vueOutputTarget } from '@stencil/vue-output-target'
 
 import { reactBooleanFixOutputTarget } from './output-target/react-boolean'
 
+// Since Ionic components are imported in `global.ts`
+// we need to exclude them here to avoid unnecessary imports in the React output module
```

Comment:
As we are using this in `vueOutputTarget` we could add Vue in the informative comment
```suggestion
// we need to exclude them here to avoid unnecessary imports in the React and Vue output module
```